packages/jest-resolve/src/__tests__/isBuiltinModule.test.ts

```diff
@@ -23,4 +23,8 @@ describe('isBuiltinModule', () => {
   it('should return false for any internal node builtins', () => {
     expect(isBuiltinModule('internal/http')).toBe(false);
   });
+
+  it('should return true for the `node:http` module', () => {
```

Comment:
We should probably also test that `node:not-a-code-module` returns `false`.

Reply:
Reasonable. I'll add this case, thanks.

Reply:
Hmm... `isBuiltinModule` lays deeper than `isCoreModule`, and imho the fix should be applied as much closer to the point where the issue raises. Anyway, I'd be happy if any of our PRs would have merged as soon as possible.

Reply:
> Anyway, I'd be happy if any of our PRs would have merged as soon as possible.

Yes, hopefully, one of our PR will get merged soon.